chroma-manager/Makefile

```diff
@@ -74,10 +74,12 @@ version:
 	echo 'IS_RELEASE = $(IS_RELEASE)' >> scm_version.py
 
 develop: version ui-modules
-	$(MAKE) -C ../chroma-agent develop
-	$(MAKE) -C ../cluster-sim develop
-	python setup.py develop
-	./manage.py dev_setup $(DEV_SETUP_BUNDLES)
+	export PYTHONPATH=$$(pwd)/../chroma-common:$$PYTHONPATH; \
```

Comment:
This block should start with a `set -e` and with that, the `&&`s can be changed to `;`.

Should also line up the line continuators.